modules/graphql_views/src/Plugin/GraphQL/Fields/View.php

```diff
@@ -59,6 +62,22 @@ public static function create(ContainerInterface $container, array $configuratio
   /**
    * {@inheritdoc}
    */
+  public function getCacheDependencies($result, $value, array $args) {
+    if (isset($result)) {
+      $metadata = new CacheableMetadata();
+      foreach ($result as $row) {
+        if ($row instanceof CacheableDependencyInterface || $row instanceof ViewExecutable) {
+          $metadata->addCacheTags($row->getCacheTags());
+        }
+      }
+      return [$metadata, $value, $args];
```

Comment:
Sorry, please just return $metadata here. My bad. We found a bug and when we solved it we also agreed to only return $result in the default implementation.